css-fonts/styles.css

```diff
@@ -0,0 +1,13 @@
+body {
+  line-height: 1.6em;
+}
+
+body p {
+  font-family: 'Fira Sans', sans-serif;
+}
+
+body
```

Comment:
Kinda similar to above, you don't need to target the `body` here, you can just target the `h1` & `h2`. This is related to something called CSS Cascade which we will touch on in a later exercise.